docs/classic-ui/icons.md

```diff
@@ -35,99 +35,112 @@ See the file [package.json](https://github.com/plone/plone.staticresources/blob/
+- [Country Flags](https://github.com/plone/plone.staticresources/blob/master/src/plone/staticresources/profiles/default/registry/icons_country_flags.xml)
+- [Plone](https://github.com/plone/plone.staticresources/blob/master/src/plone/staticresources/profiles/default/registry/icons_plone.xml)
+- [Toolbar](https://github.com/plone/plone.staticresources/blob/master/src/plone/staticresources/profiles/default/registry/icons_plone.xml)
+
+The icons above are made available as plone resources.
+For example, the lightning icon's resource path is `++plone++bootstrap-icons/lightning.svg`. One could register another icon set under a new name, for example `++plone++fontawesome-icons` and override the registrations above to use them.
 
 
 (classic-ui-icons-contextual-icons-label)=
 
 ## Contextual Icons
 
-We can define contextual icons to be used in specific places in a website while preserving the default system icons to be used elsewhere throughout the site.
-To use a different icon than the system default, you would override it without touching the system icon.
- 
-You should avoid replacing system icons because it may result in an inconsistent user interface.
-
-The specific icon name is based on the usage in the system.
-For example the copy icon is named `plone.icon.plone-copy`.
+Plone defines contextual icons, which are used in specific places in a website.
+For example we have an icon registered under the name `plone.icon.plone-copy` in https://github.com/plone/plone.staticresources/blob/master/src/plone/staticresources/profiles/default/registry/icons_plone.xml which points to the bootstrap clipboard-plus icon `++plone++bootstrap-icons/clipboard-plus.svg`.
+To use a different icon than the system default, you can override the registration for `plone.icon.plone-copy` with another icon path.
 
 
 (classic-ui-icons-icon-expression-label)=
 
 ## Icon expression
 
+```{note}
+How does this work? We need an example here!
+```
+
 - The field `icon_expression` is used again to define icons for actions, contenttypes, and other purposes.
 - Use the icon name for icon expressions.
 
 
-(classic-ui-icons-customization-label)=
-
-## Customization
-
-```{todo}
-Add content for this section
-```
-
-- Add custom icons
-- Override some of the icons
-
+(classic-ui-icons-iconresolver-label)=
 
-(classic-ui-icons-icon-font-label)=
+## Iconresolver
 
-## Icon font
+The iconresolver take's an icon name (without the `plone.icon.` prefix) like `plone-copy` or `align-center` and resolves it to an actual icon.
 
-```{todo}
-Add content for this section
+```
+http://localhost:8080/Plone/@@iconresolver/plone-copy
 ```
 
-- Set up a profile to install the icon font.
-- Use Font Awesome as an example.
-
-
-(classic-ui-icons-iconresolver-label)=
-
-## Iconresolver
-
-```{todo}
-What is `iconresolver`?
-Add to Glossary.
 ```
+http://localhost:8080/Plone/@@iconresolver/align-center
+```
+
+It can be used via URL or via Python as shown below.
```

Comment:
This is not clear. What is "it"? The icon resolver or the URL?

Which section below? Please name the specific section or use a link, e.g.:

```
...as described in the section {ref}`classic-ui-icons-iconresolver-get-icon-url-label`.
```